### PR TITLE
Get and set db password

### LINKED ISSUE #140: Add ability to set/get the DB password
create command `mu db get-password` to return the current password from parameter store

create command `mu db set-password` to prompt for a new password to set into parameter store

### PR DESCRIPTION
**Pull request by juddmon**
Add ability to set and get a database password.  This resolves #140

### CODE REVIEW

File: cli/app.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ func NewApp() *cli.App {
 		*newServicesCommand(context),
 		*newPipelinesCommand(context),
 		*newDatabasesCommand(context),
+		*newDatabaseGetPasswordCommand(context),
```
Comment (cplee) [inline cli/app.go]: these should be sub commands in the database.go file

File: workflows/database_params.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+// DatabaseSetPassword sets a database password for an environment
+func DatabaseSetPassword(ctx *common.Context, environmentName string, serviceNameIn string, newPassword string) Executor {
+	return func() error {
+		workflow := new(databaseWorkflow)
+		workflow.codeRevision = ctx.Config.Repo.Revision
+		workflow.repoName = ctx.Config.Repo.Slug
+		serviceName := serviceNameIn
+		if serviceName == "" {
+			serviceName = workflow.serviceName
```
Comment (cplee) [inline workflows/database_params.go]: Does this work?  this doesn't appear to be initialized anywhere.

Also, the workflow is being created by not executed...this isn't following the same pattern as other commands.

File: workflows/database_params.go [hunk 3]
```diff
@@ -6,16 +6,8 @@ import (
 )
 
 // DatabaseSetPassword sets a database password for an environment
-func DatabaseSetPassword(ctx *common.Context, environmentName string, serviceNameIn string, newPassword string) Executor {
+func DatabaseSetPassword(ctx *common.Context, environmentName string, serviceName string, newPassword string) Executor {
```
Comment (cplee) [inline workflows/database_params.go]: How are you defaulting `serviceName` to the one in mu.yml if not provided on CLI?

Comment (juddmon) [reply workflows/database_params.go]: I added the workflow back in and used database_common to use the serviceName from mu.yml.

**Pull request merged**